client/app/dedicatedCloud/user/dedicatedCloud-user.html

```diff
@@ -73,6 +73,11 @@ <h2 data-i18n-static="dedicatedCloud_user_management_authorization_vsphere"></h2
                         class="text-center"
                         data-i18n-static="dedicatedCloud_USER_ipfo">
                     </th>
+                    <th scope="col"
```

Comment:
You should update the `colspan` value as well. (for those empty `<tbody>` elements :))